.claude/skills/mastrabowl-recap/template.md

```
@@ -0,0 +1,71 @@
+# Dear ____,
+
+[song]
+
+### Highlights
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)